examples/lib/stories/experimental/layout_component_example.dart

```diff
@@ -36,8 +48,9 @@ layouts to reflect the chosen values.
           return ButtonComponent(
```

Comment:
I think all these buttons should be kept in the settings section now, since the root component is now so large that it is hard to get an overview, and with the current button implementation you also can't see which settings that are active, you can take inspiration from [here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/rendering/rendering.dart#L56-L67) (it should be fairly easy to change this since most of it is built-in to dashbook).

Reply:
Oh wow, I didn't even know this existed. I played around with it, and I'm happy to implement it, but I have 2 concerns:

1. It seems to simply not work with `GameWidget.controlled`. Perhaps I'm using it wrong? But so far I do not see any invocation of `GameWidget.controlled` that also uses dashbook's property controllers.
2. It seems to follow Flutter's paradigm, such that when you change a property, it reconstructs the widget. This is well and good, but with this model, we lose out on the ability to see what happens when you change the property of a component directly, vs constructing it anew. And I can't see an escape hatch in dashbook that lets me use a property in the "flamey" way.

Reply:
@erickzanardo would it be possible to make so that we can change mutable variables directly, and not change the flutter state in dashbook?

Reply:
> It shouldn't be a problem to use the "conventional" properties, the rebuilds should maintain the state of the game widget shouldn't it?

The problem is that when using `controlled` it won't rebuild the `GameWidget`, and when not using `controlled` then the resizing of the window is janky, which is something that one might want to try out on this example.

Reply:
Oops @spydon I may have spoken too soon. Somehow I'm getting a stack overflow on my existing project. Migrating to the new API you designed doesn't seem to be as straightforward.